modules/efk-logging-fluentd-json.adoc

```diff
@@ -32,3 +32,37 @@ oc set env dd/fluentd MERGE_JSON_LOG=true <1>
 ----
 
 <1> Set this to `false` to disable this feature or `true` to enable this feature.
+
+[id="efk-logging-fluentd-json-string-{context}"]
+== Setting MERGE_JSON_LOG and CDM_UNDEFINED_TO_STRING
+
+You can use link:https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/README.md[environment variables] to modify your fluentd configuration.
+
+However, if you set the `CDM_UNDEFINED_TO_STRING` enviroment variable to `true` when `MERGE_JSON_LOG` is also set to `true`, you will receive an Elasticsearch *400* error if you have already generated conflicting fields, until the indices roll over for the next day. The error occurs because when`MERGE_JSON_LOG=true`, Fluentd adds fields with data types other than *string*. When you set `CDM_UNDEFINED_TO_STRING=true`, Fluentd attempts to add those fields with a *string* value resulting in the Elasticsearch *400* error. 
+
+When Fluentd rolls over the indices for the next day's logs, it will create a brand new index. The field definitions are updated and you will not get the *400* error. The only thing that can be done is to use something like https://github.com/richm/docs/releases/tag/20180510180102 - the section about @ERROR Error record handling:
```

Comment:
I don't think you want to put a link to my blog in the official product docs.
maybe have a link to https://docs.fluentd.org/v1.0/articles/config-file#@error-label instead.